src/language/functions.md

```diff
@@ -500,6 +500,26 @@ Iterable<int> naturalsDownFrom(int n) sync* {
 }
 ```
 
+## External functions {#external}
+
+Declare a function `external` if its body is defined separately from its
+declaration in your Dart program. 
```

Comment:
This sentence being imperative makes me slightly concerned developers will misinterpret it as "if you make a function `external` in one file, **you** can implement it in another **Dart** file". Is that worry justified?

If so, would it make sense to instead just mention _how_ they are declared and rely on the various interop docs for actual instructions? Or move up the discussion from the last paragraph about when it appears, since that's the main reason we're documenting it? Not sure 😕 

Maybe I'm overthinking it. I'll think some more but will leave this comment for now to get your thoughts :)

Reply:
No that's a really good point (in fact, it sounds like exactly the kind of question I'd leave in a review of someone else's work haha).

I do want to leave the last paragraph where it is (though your reasoning makes sense, I think since this is purely a reference section the first sentence should be a generic definition), but I think I'll restate the opening more or less as it's stated in the [spec](https://spec.dart.dev/DartLangSpecDraft.pdf#External%20Functions):

> An external function is a function whose body is provided separately from its declaration.